CODINGSTYLE.md

```diff
@@ -0,0 +1,42 @@
+# prplMesh coding style
+
+This document describes the intended coding style.
+Note that existing code does not always comply to it.
+
+Line length is 100 characters.
```

Comment:
We should allow some exceptions - for example tables and anything that looks hideous in 100 column boundary can skip clang-format by adding `clang-format-off` and `clang-format-on` surrounding it.